ring-jetty-adapter/test/ring/adapter/test/jetty.clj

```diff
@@ -35,6 +36,12 @@
 (defn- all-threads []
   (.keySet (Thread/getAllStackTraces)))
 
+(defn ssl-context
+  []
```

Comment:
As a small point, could you put the `[]` on the previous line?

Reply:
done ;)